### PR TITLE
V3.21: FiYir Changes for Arena Open Support

### PR DESCRIPTION
**Pull request by unrealities**
* 3.21 Bump
* Update Python Libraries
* FiYir Arena Open Support README update

### CODE REVIEW

File: requirements.txt
Comment (FiYir) [inline requirements.txt:1]: Could you explain the "Excluded if auto-generated" comment? Are you planning on removing both of these packages?

Comment (unrealities) [reply requirements.txt:1]: I modified the comment. I was trying to update the dependencies using `pipreqs,` which removed the two lines below the comment. I don't know Python standards, as these aren't part of the core project but are necessary for compiling a Windows build.

Comment (FiYir) [reply requirements.txt:1]: You can remove those two packages from the requirements file and add them to the build-windows-exe [YAML file](https://github.com/unrealities/MTGA_Draft_17Lands/blob/main/.github/workflows/build-windows-exe.yml)

image

Comment (unrealities) [reply requirements.txt:1]: That makes more sense. Thank you.

**Comment by FiYir** [comment]
Hey, could you update the release notes to say "FiYir added support for the Arena Open, Qualifier, and Arena Direct events"? I think the other notes from my pull request can be skipped. Just add the links for #23 and #26 to the release page.

Also, any plans to release this today? JaggerMin and others might use it for the Qualifier tomorrow, and I'm curious to see if it's working.

**Pull request merged**